.travis.yml

```diff
@@ -7,9 +7,9 @@ cache:
   yarn: true
 
 script:
-  - yarn run lint
-  - yarn run build
-  - yarn run cover
+  - npm run lint
```

Comment:
We experienced build errors with yarn on Travis today. Also, quoting @felixfbecker : 

> I don’t want to check in yarn.lock, because this is a package, not a project. if we check in yarn.lock it gives a false sense of security - the build can pass, but a user that `npm install`s the package could get a broken package